sdk/communication/azure-communication-chat/README.md

```diff
@@ -478,12 +478,12 @@ if retry:
 ### Remove thread participant
 
 Use `remove_participant` method to remove thread participant from the thread identified by threadId.
-`user` is the `CommunicationUserIdentifier` you created by CommunicationIdentityClient.create_user() from User Access Tokens
+`identifier` is the `CommunicationUserIdentifier` you created by CommunicationIdentityClient.create_user() from User Access Tokens
```

Comment:
```suggestion
`identifier` is the `CommunicationUserIdentifier` you created by CommunicationIdentityClient.create_user() from azure-communication-identity
```

Reply:
The User Access Token was originally present as an anchor which had been removed. Cleaned it up and changed it to communication-identity